src/data/polynomial/degree/basic.lean

```diff
@@ -268,6 +268,41 @@ by simpa only [C_1, one_mul, pow_one] using degree_C_mul_X_pow_le (1:R) 1
 lemma nat_degree_X_le : (X : polynomial R).nat_degree ≤ 1 :=
 nat_degree_le_of_degree_le degree_X_le
 
+lemma support_mul_X_pow (c : R) (n : ℕ) (H : c ≠ 0) : (C c * X^n).support = singleton n :=
```

Comment:
These lemmas don't have anything to do with `degree`, right? So I would suggest moving them to `polynomial/basic.lean`.

Reply:
@adomani Could you try moving them again? In VScode, cut them from this file, paste them in the other file; save both files; commit; push. That should do it.